network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube_regrets_page.html

```diff
@@ -86,5 +86,5 @@
 {% endblock %}
 
 {% block footer_block %}
-{% include "partials/footer.html" with skip_donate=True %}
+{% include "partials/footer.html" with skip_donate=False %}
```

Comment:
due to https://github.com/mozilla/foundation.mozilla.org/blob/master/network-api/networkapi/templates/partials/footer.html#L5 we can just do this:
```suggestion
{% include "partials/footer.html" %}
```